synapse/lib/storm.py

```diff
@@ -1131,9 +1137,7 @@ async def getTeleProxy(self, url, **opts):
         return prox
 
     def isRuntVar(self, name):
-        if name in self.runtvars:
-            return True
-        return False
+        return bool(self.runtvars.get(name))
```

Comment:
if the name is present in self.runtvars and the value is False, this will return False

Reply:
Would be helpful to note this dictionary is a map of vars to their runt safety for future selves?

Reply:
as intended. this is just to ensure we return False rather than None if it's not present ( and therefor not runtsafe :D )